internal/client/main.go

```diff
@@ -18,6 +18,10 @@ type Client struct {
 // with provider configuration.
 func New(bearer string, organization string) *Client {
 	cfg := openapi.NewConfiguration()
+	
+	// Add default headers to the configuration
+	cfg.AddDefaultHeader("Authorization", "Bearer "+bearer)
```

Comment:
@steveworley in some of the API calls made (specifically crawler add/read) there was no bearer token added, so have added here for now which resolves

Reply:
This might fix the issue with custom headers too :D